src/Sulu/Bundle/PreviewBundle/Domain/Model/PreviewLink.php

```diff
@@ -53,15 +53,6 @@ class PreviewLink implements PreviewLinkInterface
      */
     private $lastVisit;
 
```

Comment:
Would still keep the constructor this way:

```suggestion
    public function __construct(string $token, string $resourceKey, string $resourceId, string $locale, array $options)
    {
        $this->token = $token;
        $this->resourceKey = $resourceKey;
        $this->resourceId = $resourceId;
        $this->locale = $locale;
        $this->options = $options;
    }
```